src/com/lftechnology/java/training/alina/jdbc/service/UserRoleService.java

```diff
@@ -0,0 +1,177 @@
+package com.lftechnology.java.training.alina.jdbc.service;
+
+import java.sql.SQLException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import com.lftechnology.java.training.alina.jdbc.constants.Constants;
+import com.lftechnology.java.training.alina.jdbc.controller.EmployeeController;
+import com.lftechnology.java.training.alina.jdbc.controller.LoginController;
+import com.lftechnology.java.training.alina.jdbc.domain.Employee;
+
+public class UserRoleService {
+
+    private static final Logger LOGGER = Logger.getLogger(UserRoleService.class.getName());
+
+    private UserRoleService() {
+
+    }
+
+    /**
+     * Adds new employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static void addEmployee(Scanner scanner) throws SQLException {
+        LOGGER.log(Level.INFO, Constants.ADD_NEW_EMPLOYEE);
+        EmployeeController.addNewEmployee(scanner);
+    }
+
+    /**
+     * Deletes an employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     * @throws SQLException
+     */
+    public static void deleteEmployee(Scanner scanner) throws SQLException {
+        LOGGER.log(Level.INFO, Constants.DELETE_EMPLOYEE);
+        EmployeeController.deleteExistingEmployee(scanner);
+    }
+
+    /**
+     * Terminates an employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     * @throws SQLException
+     */
+    public static void terminateEmployee(Scanner scanner) throws SQLException {
+        LOGGER.log(Level.INFO, Constants.TERMINATE_EMPLOYEE);
+        EmployeeController.terminateExistingEmployee(scanner);
+    }
+
+    /**
+     * Gets employee list
+     * 
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     * @throws SQLException
+     */
+    public static void getEmployeeList() throws SQLException {
+        LOGGER.log(Level.INFO, Constants.VIEW_EMPLOYEE_LIST);
+        EmployeeController.getEmployeeList();
+    }
+
+    /**
+     * Searches an employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     * @throws SQLException
+     */
+    public static void searchEmployee(Scanner scanner) throws SQLException {
+        LOGGER.log(Level.INFO, Constants.SEARCH_EMPLOYEE);
+        EmployeeController.searchExistingEmployee(scanner);
+    }
+
+    /**
+     * Logout employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    // public static void employeeLogout(Scanner scanner) throws SQLException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)